### PR TITLE
New functions AddComment and Attachment

### PR DESCRIPTION
**Pull request by eparis**
Shamelessly stolen from the work of jeromenerf in
https://github.com/VojtechVitek/go-trello/pull/2

### CODE REVIEW

**Comment by eparis** [comment]
@jeromenerf   should get the rest of the work you did in #2, except webhooks...

**Comment by VojtechVitek** [comment]
Thanks @eparis & @jeromenerf .

File: card.go [hunk 1]
```diff
@@ -124,6 +124,20 @@ func (c *Card) Attachments() (attachments []Attachment, err error) {
...
+// Attachment will return the specified attachment on the card
+// https://developers.trello.com/advanced-reference/card#get-1-cards-card-id-or-shortlink-attachments-idattachment
+func (c *Card) Attachment(attachmentId string) (*Attachment, error) {
+	body, err := c.client.Get("/cards/" + c.Id + "/attachments/" + attachmentId)
+	if err != nil {
+		return nil, err
+	}
+
+	attachment := &Attachment{}
+	err = json.Unmarshal(body, attachment)
```
Comment (VojtechVitek) [inline card.go:136]: seems like you're ignoring this `err`

Comment (eparis) [inline card.go:136]: whoops, fixed.

**Comment by VojtechVitek** [comment]
LGTM

**Pull request merged**